learn/safe-core-account-abstraction-sdk/protocol-kit/README.md

```diff
@@ -1,7 +1,278 @@
+  value: treasuryAmount
+}]
+
+const tx = await owner1Signer.sendTransaction(transactionParameters)
+
+console.log('Fundraising.')
+console.log(`Deposit Transaction: https://goerli.etherscan.io/tx/${tx.hash}`))
+```
+
+## Making a Transaction from a Safe
+
+Owner 1 will sign and propose a transaction to send 0.05 ETH out of the Safe. Then, owner 2 will add their own proposal and execute the transaction since it meets the 2 of 3 thresholds.
+
+At a high level, making a transaction from the Safe requires the following steps:
+
+### Overview
+
+1. Owner 1 creates a transaction
+    1. Define the amount, destination, and any additional data to include in the transaction
+2. Owner 1 proposes a transaction
+    1. Performs an off-chain signature of the transaction before proposing
+    2. Submits it to the Safe Service
+3. Owner 2  gets pending transactions from the Safe service
+4. Owner 2  confirms the transaction
+    1. Performs an off-chain signature
+    2. Submits to service
+5. Anyone can execute the transaction (Owner 1 in this example)
+
+### Create a Transaction
+
+For more details on what to include in a transaction see, [Create a Transaction in the Safe Core SDK Guide](https://github.com/safe-global/safe-core-sdk/blob/main/guides/integrating-the-safe-core-sdk.md#4-create-a-transaction).
+
+```tsx
+import { SafeTransactionDataPartial } from '@safe-global/safe-core-sdk-types'
+
+// Any address can be used. In this example we will use vitalik.eth
+const destination = '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045'
+const amount = ethers.utils.parseUnits('0.05', 'ether').toString()
+
+const safeTransactionData: SafeTransactionDataPartial = {
+  to: destination,
+  data: '0x',
+  value: amount
+}
+// Create a Safe transaction with the provided parameters
+const safeTransaction = await safeSdkOwner1.createTransaction({ safeTransactionData })
+
+### Propose a Transaction
+
+```tsx
+// Deterministic hash based on transaction parameters
+const safeTxHash = await safeSdkOwner1.getTransactionHash(safeTransaction)
+
+// Sign transaction to verify that the transaction is coming from owner 1
+const senderSignature = await safeSdkOwner1.signTransactionHash(safeTxHash)
+
+await safeService.proposeTransaction({
+  safeAddress,
+  safeTransactionData: safeTransaction.data,
+  safeTxHash,
+  senderAddress,
+  senderSignature: senderSignature.data,
+  origin
+})
+```
+
+### Confirm the Transaction: First Confirmation
+
+```tsx
+// transaction: SafeMultisigTransactionResponse
+
+const hash = transaction.safeTxHash
+let signature = await safeSdkOwner1.signTransactionHash(hash)
+await safeService.confirmTransaction(hash, signature.data)
+```
+
+### Get Pending Transactions
+
+Owner 2 needs a different Safe object. However, you don’t need to create it with the safe factory you can create it with the `create` method of the Safe object. The Safe smart contract is already live on the blockchain so you can just pass the treasury address used when you created the Safe.
```

Comment:
```suggestion
When owner 2 is connected to the application, the Safe Core SDK should be initialized again with the existing Safe address the address of the owner 2 instead of the owner 1.
```

Reply:
This paragraph belongs to the `Confirm the Transaction: Second Confirmation` section. Get pending transaction only relies on the Safe Service Client.

Reply:
Fixed here: https://github.com/safe-global/safe-docs/pull/165/commits/360439d53c465d9e0377a9887df012a6644f9a14

Reply:
For some reason this change was not applied. Maybe the code was moved before applying the change ;)